client/src/app/components/Header.scss

```diff
@@ -1,188 +1,383 @@
-@import '../../scss/mixins';
-@import '../../scss/variables';
+@import "../../scss/fonts";
+@import "../../scss/media-queries";
+@import "../../scss/mixins";
+@import "../../scss/variables";
+@import "../../scss/themes";
 
 header {
   display: flex;
-  position: relative;
+  position: fixed;
+  z-index: 1000;
   justify-content: center;
-  padding: 15px 20px;
-  background-color: var(--context-primary);
+  width: 100%;
+  background: var(--navbar-bg);
+  backdrop-filter: blur(20px);
+
+  a {
+    outline: none;
+    text-decoration: none;
+  }
 
   .inner {
     display: flex;
     flex: 1;
-    flex-wrap: wrap;
+    flex-direction: row;
     align-items: center;
     justify-content: space-between;
-    max-width: $desktop-width;
+    height: $navbar-height-desktop;
 
-    @include tablet-down {
-      margin-right: 40px;
+    @include desktop-down {
+      flex-direction: column;
+      flex-wrap: wrap;
+      height: calc(2 * #{$navbar-height-mobile});
     }
 
-    @include phone-down {
-      margin-right: 0;
+    .logo-image--wrapper {
+      display: flex;
+      align-items: center;
+
+      svg {
+        width: auto;
+        height: 48px;
+
+        @include desktop-down {
+          height: 40px;
+        }
+
+        path {
+          fill: var(--body-color);
+        }
+      }
     }
 
-    a {
-      margin-right: 30px;
-      outline: none;
+    .inner--main,
+    .inner--networks {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      height: $navbar-height-desktop;
+      padding: 16px 0;
+      border-bottom: 1px solid var(--border-color);
 
-      &:focus {
-        box-shadow: 0 0 3px 0 $main-green-highlight;
+      @include desktop-down {
+        width: 100%;
+        height: $navbar-height-mobile;
       }
     }
 
-    .search-input {
-      margin: 10px 30px 10px 5px;
+    .inner--main {
+      flex: 1;
+      padding: 0 14px 0 24px;
 
-      @include phone-down {
-        margin: 10px 30px 15px 0;
+      .mobile-fiat {
+        display: none;
+      }
+
+      @include desktop-down {
+        .mobile-fiat {
+          display: block;
+          margin-left: auto;
+        }
+
+        .desktop-fiat {
+          display: none;
+        }
       }
     }
 
-    .logo-image {
-      width: 210px;
-      height: 48px;
+    .inner--networks {
+      flex-shrink: 1;
+      padding-right: 24px;
+
+      @include desktop-down {
+        padding: 0 24px;
+      }
     }
-  }
 
-  .tools {
-    &.tools--large {
-      @include font-size(14px);
+    .navigation--item,
+    .utilities--wrapper {
+      @include font-size(14px, 21px);
 
       display: flex;
-      position: absolute;
-      top: 0;
-      right: 0;
       align-items: center;
-      justify-content: center;
-      width: 150px;
       height: 100%;
-      border: 0;
-      border-radius: 0;
-      outline: 0;
-      background: rgba(72, 87, 118, 0.1);
-      color: $white;
+      margin-left: 40px;
+      color: $gray-6;
```

Comment:
Change rule to color: var(--header-color);

Reply:
Resolved

Reply:
Changed to color: var(--navbar-color);